frontend/src/components/ViewTrips/trips-container.js

```diff
@@ -32,12 +32,13 @@ function getErrorElement(error) {
  * Component corresponding to the container containing a user's trips.
  * props
  *
- * @param {Object} props These are the props for this component:
- * - handleEditTrip: Handler that displays the edit trip modal.
+ * @property {Object} props These are the props for this component:
+ * @property {Function} props.handleEditTrip Event handler responsible for
+ *     displaying the edit trip modal.
  * @extends React.Component
```

Comment:
ooh, I missed @extends! That'd be good to have.

Reply:
Yeee ye